### ISSUE TITLE
listing v6-ranges does not indicate prefix length

### ISSUE THREAD

**Issue by nivex** [linode/linode-cli#207]
```
~$ linode-cli --version
linode-cli 3.1.0
Built off spec version 4.72.0
```

When listing `v6-pools`, the prefix length is shown:
```
~$ linode-cli networking v6-pools
┌────────────────────┬────────┬─────────┐
│ range              │ prefix │ region  │
├────────────────────┼────────┼─────────┤
│ 2600:3c03::1b:1000 │ 116    │ us-east │
└────────────────────┴────────┴─────────┘
```

However, when listing `v6-ranges`, the prefix length is not shown:
```
~$ linode-cli networking v6-ranges
┌───────────────────────┬─────────┐
│ range                 │ region  │
├───────────────────────┼─────────┤
│ 2600:3c03:e001:0600:: │ us-east │
└───────────────────────┴─────────┘
```

For consistency, both outputs should contain the prefix length.

**Comment by Dorthu** [comment]
Thanks for the report!  I'm looking into this now.

**Comment by Dorthu** [comment]
I opened the above PR to resolve this issue.  It should be fixed in the next scheduled CLI release, which will be on 2020-09-07

**Comment by Dorthu** [comment]
This was released in version 3.1.1.  Thank you again for the report!

**Comment by nivex** [comment]
Confirmed working here. Thanks for the quick turnaround.
